grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1641,13 +1641,9 @@ def section_hummock(panels):
             ],
         ),
         panels.timeseries_percentage(
-            "Filter/Cache Miss Rate",
+            "Cache Miss Rate",
             "",
             [
-                panels.target(
-                    f"1 - (sum(rate({metric('state_store_bloom_filter_true_negative_counts')}[$__rate_interval])) by (job,instance,table_id,type)) / (sum(rate({metric('state_bloom_filter_check_counts')}[$__rate_interval])) by (job,instance,table_id,type))",
-                    "bloom filter miss rate - {{table_id}} - {{type}} @ {{job}} @ {{instance}}",
-                ),
```

Comment:
This metrics has no data.

Reply:
Missing `bloom_filter_check_counts` is reported, instead of delete the panel. @Little-Wallace